src/index.d.ts

```diff
@@ -222,11 +222,11 @@ declare namespace preact {
 		vnode: ComponentChild,
 		parent: Element | Document | ShadowRoot | DocumentFragment
 	): void;
-	function cloneElement(
-		vnode: JSX.Element,
+	function cloneElement<P>(
```

Comment:
Because of the nature of this file (no ts-compiler) you'll have to specify the non-generic version as well (comparable to createElement https://github.com/preactjs/preact/pull/2388/files#diff-b739f95ace6bbf2f0fd5e8cf1dab97d8R176)

Reply:
Thanks for the quick feedback! 😄

Reply:
Shall we consider adding a test for the generic version?

Reply:
Works for me yes! Would be awesome if you did